packages/yarnpkg-sdks/sources/sdks/base.ts

```diff
@@ -16,11 +17,14 @@ export const generateAstroLanguageServerBaseWrapper: GenerateBaseWrapper = async
 export const generateEslintBaseWrapper: GenerateBaseWrapper = async (pnpApi: PnpApi, target: PortablePath) => {
   const wrapper = new Wrapper(`eslint` as PortablePath, {pnpApi, target});
 
-  await wrapper.writeManifest();
+  const pkg = await wrapper.writeManifest();
 
   await wrapper.writeBinary(`bin/eslint.js` as PortablePath);
   await wrapper.writeFile(`lib/api.js` as PortablePath, {requirePath: `` as PortablePath});
 
+  if (semverUtils.satisfiesWithPrereleases(pkg.version, `>=8`))
```

Comment:
Shouldn't this be `>=8.0.0-sdk` since `writeManifest` appends `-sdk` to the version which would make it a prerelease, so this would only end up matching for eslint 8.0.1 and up ?